schedule/models/events.py

```diff
@@ -461,11 +461,8 @@ def __unicode__(self):
             'end': self.end,
         }
 
-    def __cmp__(self, other):
-        rank = cmp(self.start, other.start)
-        if rank == 0:
-            return cmp(self.end, other.end)
-        return rank
+    def __lt__(self, other):
+        return self.start < other.start
```

Comment:
it's not isofunctionnal without end comparison ?

Reply:
```
if self.start < other.start or (self.start == other.start and self.end < other.end):
    return True
return False
```

Better?

Reply:
yes, but __cmp__ in python 2 used to combine __lt__, __eq__ and __gt__

do you know where's the comparison between events take place to be sure that we don't have to implement __eq__ and __gt__ ?

Reply:
Fixed, commit --amend and push -f